rc/windowing/tilix.kak

```diff
@@ -0,0 +1,69 @@
+
+provide-module tilix %{
+
+# ensure that we're running on kitty
```

Comment:
I'm assuming this is copied from the kitty support script? This should probably be changed to tilix instead.

Reply:
Indeed. If you wish to make changes feel free to push on my fork; I wish I had time to improve this and merge arrufat's version, but it seems to be hard atm :(